project_stock/models/account_analytic_line.py

```diff
@@ -9,6 +9,9 @@ class AccountAnalyticLine(models.Model):
     stock_task_id = fields.Many2one(
         comodel_name="project.task", string="Project Task", ondelete="cascade"
     )
+    stock_move_id = fields.Many2one(
+        comodel_name="stock.move", string="Stock Move", ondelete="cascade", copy=False
```

Comment:
Adding this field now will make the previous lines orphaned. Aren't you able to do it without this field?